.travis.yml

```diff
@@ -8,11 +8,18 @@ python:
  - "3.4"
  - "3.5"
  - "pypy"
+ - "pypy3"
+ - "pypy3.3-5.2-alpha1"
  - "nightly"
-install: pip install .
-script: tox
-branches:
-  only:
-    - master
-matrix:
-  fast_finish: true
+
+before_install:
+ # Show the current setuptools version
+ - ORIG_SETUPTOOLS_VERSION=`python -c "import setuptools; print('%s' % setuptools.__version__)"`
+ - echo setuptools $ORIG_SETUPTOOLS_VERSION
```

Comment:
I am guessing that is new, and probably not in the Travis pip version yet. (14 days ago it was 6.0.7, and I doubt that has changed.)

```bash
$ pip --version
pip 8.0.2 from /usr/lib/python2.7/site-packages (python 2.7)
 pip freeze --all | grep setuptools

Usage:   
  pip freeze [options]

no such option: --all
$ pip freeze | grep setuptools
You are using pip version 8.0.2, however version 8.1.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```